docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,165 @@
+   ```
+
+   The preceding command is necessary only if you are currently in manifests update mode. If you don't know, [check the current update mode](#check-the-update-mode-and-version). Workload-set update mode is the default starting in .NET 9 SDK.
+
+1. ```dotnetcli
+   dotnet workload update
+   ```
+
+   In workload set update mode, this command updates workloads to the latest workload set version, unless you have specified the workload set version in global.json.
+
+## Update to a workload-set version
+
+To specify a workload set version to update to when you're not specifying it in global.json, use the `--version` option of the `update` command. For example:
+
+```dotnetcli
+dotnet workload update --version 8.0.400
+```
+
+Workload-set update mode will be selected if it wasn't already selected.
+
+## Use global.json for the workload set version
+
+Create a `global.json` file to specify the workload set version for a repository. The format looks like this example:
+
+```json
+{
+  "sdk": {
+    "workloadVersion": "9.0.100-preview.7.24414.1"
+  }
+}
+```
+
+With the current directory in the same repository and the CLI in workload set update mode, the `install`, `update`, and `restore` commands install workloads for the specified workload set version. If you don't have a global.json file, and you're in workload set update mode, the `restore` command installs the workload set version that was established when you switched from manifests update mode to workload sets update mode.
+<!--todo:what about if it's SDK 9 so you defaulted to workload set update mode, and you never established a workload set version -- does it go for latest available then?-->
+<!--todo:if you're in manifests update mode, you stay in that mode and it doesn't seem to do a workload set update, i.e., it seems to ignore global.json(?)-->
+
+If you specify the workload set version in global.json, you can't use the `--version` option to specify the workload set version. To use the `--version` option in that case, run the command outside of the path containing the global.json file.
+
+If you don't specify the workload set version in global.json, you can use the `--version` option with the `restore` comand. In that case, the `restore` command selects workload set update mode before it restores workloads to the specified workload set version.
+
+In manifests update mode, `restore` installs or updates workloads to the latest version of each individual workload.
+
+## Check the update mode and version
+
+To see the current update mode, run the `config` command with the `--update-mode` option without an argument. The mode is either workload-set` or `manifests`. For example:
+
+```dotnetcli
+dotnet workload config --update-mode
+```
+
+```output
+workload-set
+```
+
+To see the current workload set version, run `dotnet workload --version`.  If a workload set is installed, you see a version such as 9.0.100-preview.7.24414.1 or 8.0.401. For example:
```

Comment:
fyi, there is no 8.0.401 workload set. @baronfel should we have shipped an 8.0.401 that had no changes in it from 8.0.400 since we did a hotfix for that sdk?

Reply:
fwiw, there will be an 8.0.402 workload set

Reply:
```suggestion
To see the current workload set version, run `dotnet workload --version`.  If a workload set is installed, you see a version such as 9.0.100-preview.7.24414.1 or 8.0.402. For example:
```